Classes/TreeTab.lua

```diff
@@ -357,4 +381,22 @@ function TreeTabClass:OpenExportPopup()
 		main:ClosePopup()
 	end)
 	popup = main:OpenPopup(380, 100, "Export Tree", controls, "done", "edit")
+end
+
+function TreeTabClass:updatePower()
+	s = {
+		{ height = 20, "^7Node power list, descending (" .. self.controls.treeHeatMapStatSelect.list[self.controls.treeHeatMapStatSelect.selIndex].label .. "):" },
+	}
+    if (self.controls.treeHeatMapStatSelect.selIndex > 1) then
+        table.sort(nodePowers, function(a,b) return a.power > b.power end)
+        for _,tableitem in ipairs(nodePowers) do
+            t_insert(s, { height = 16, "^7" .. tableitem.text .. tableitem.power })
+            t_insert(s, {height = 16, ""})
+        end
+	else
+		t_insert(s, {height = 25, "Cannot process Offence/Defence!"})
```

Comment:
What's this for?

Reply:
Since the offence/defence highlight is not claculated using "node.power.singleStat" I would have to write alot more code to make it work correctly with the current Node List implementation, if it is even possible. And since all people I know who use PoB say they almost never use offence/defence highlight, I decided to only give this message to let the user know it is not supported